tools/quicktype-wrapper/README.md

```diff
@@ -29,11 +29,16 @@ npm run start
 
 To configure the script, use _environment variables_ or _command-line flags_:
 
+- (REQUIRED) `IN`: Input directory path
+- (REQUIRED) `OUT`: Output directory path
+- (NOT REQUIRED) `NO_LICENSE`: Adds a license header to source files
```

Comment:
Why need it to be optional?

Reply:
Was thinking that some languages might want to add their own license headers or add them in a different way. Opt-out sounds fine for that.